src/CMakeLists.txt

```diff
@@ -474,11 +474,13 @@ if(FLB_BINARY)
     COMPONENT binary
     RENAME "${FLB_OUT_NAME}.conf")
 
-  install(FILES
-    "${PROJECT_SOURCE_DIR}/conf/fluent-bit-win32.conf"
-    DESTINATION ${FLB_INSTALL_CONFDIR}
-    COMPONENT binary
-    RENAME "${FLB_OUT_NAME}-win32.conf")
+  if(FLB_SYSTEM_WINDOWS)
+    install(FILES
+      "${PROJECT_SOURCE_DIR}/conf/fluent-bit-win32.conf"
```

Comment:
Is there a reason on why this configuration can't be the default on windows systems? ${FLB_OUT_NAME}.conf?